drivers/bridge/bridge.go

```diff
@@ -46,6 +42,8 @@ type Configuration struct {
 	EnableICC             bool
 	EnableIPForwarding    bool
 	AllowNonDefaultBridge bool
+	// Driver components
+	PortAllocator *portallocator.PortAllocator
```

Comment:
Agree the long term solution to have port allocator be in sync with the OS is great. When that happens, we do not need singleton, any pa instance will be in sync with the OS. 

But now, from client point of view, we can provide the exactly same behavior converting portallocator to a singleton.

The nice part of this is that once we move to the long term solution, all changes will be contained in portallocator package and no API change, so that actual pa clients will not see any change in behavior.

Thanks for the comments, will publish the change soon.